### PR TITLE
1 docu in GitHub pages

### PR DESCRIPTION
**Pull request by windisch**
This PR adds a GitHub action to deploy our docs at a GitHub pages.

### CODE REVIEW

File: .github/workflows/test_docs.yml [hunk 1]
```diff
@@ -10,20 +10,18 @@ jobs:
...
-        python-version: ["3.10"]
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python ${{ matrix.python-version }}
+    - name: Set up Python
       uses: actions/setup-python@v3
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: "3.10"
```
Comment (MartinRoth) [inline .github/workflows/test_docs.yml:19]: Why do you revert from matrix - I think having the python version on top is nice - but no deal braker

**Review by MartinRoth — APPROVED** [review]
lgtm - only one small comment

File: .github/workflows/test_docs.yml (hunk 1 above)
Comment (MartinRoth) [reply .github/workflows/test_docs.yml:19]: > 

Fixes also #5

Comment (windisch) [reply .github/workflows/test_docs.yml:19]: It doesn't feel right to have the matrix-boilerplate for a `1x1` matrix.

**Pull request merged**